compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -499,6 +500,31 @@ internal class RootNodeOwner(
     }
 }
 
+// the max value that can be passed as Constraints(0, LargeDimension, 0, LargeDimension)
+// greater values cause "Can't represent a width of"
+internal const val LargeDimension = (1 shl 14) - 1 // 16383
```

Comment:
I see that in 1.6.1 merge this value is `(1 shl 15) - 1` why is it different?

Reply:
If you pass `Constraints(0, (1 shl 15) - 1, 0, (1 shl 15) - 1)`, you will get an exception. In the CL it was used only for one side